extensions/interactions/LogicProof/LogicProof.js

```diff
@@ -49,22 +49,38 @@ oppia.directive('oppiaInteractiveLogicProof', [
         );
         $scope.questionData.language.operators = $scope.typing[0].operators;
 
-        $scope.assumptionsString = logicProofShared.displayExpressionArray(
-          $scope.questionData.assumptions,
-          $scope.questionData.language.operators);
-        $scope.targetString = logicProofShared.displayExpression(
-          $scope.questionData.results[0],
-          $scope.questionData.language.operators);
+        $scope.displayExpression = function(exp) {
```

Comment:
Hm, the reason they merit comments is because I read them and I didn't understand what they do. I don't mind whether they're pulled out or left in, but if they're their own separate things then I would add a docstring.